src/components/Builder/Builder.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+export default function Builder({ setHead, setMiddle, setBottom, setCatchphrase, setCatchphraseArray, head, middle, bottom, catchphrase, catchphraseArray }) {
+
+  const saveCatchphrase = () => {
+    setCatchphraseArray((prevState) => [...prevState, catchphrase]);
+  };
```

Comment:
On the saveCatchphrase function, we can reset the input field to an empty string, using setCatchphrase(''), so that way the user doesn't have to delete what they had already typed in the input field. Definitely not a requirement here though.